rest-server/test/job.js

```diff
@@ -52,36 +63,43 @@ describe('Jobs API /api/v1/jobs', () => {
     ],
   };
 
-  // Mock launcher webservice
-  beforeEach(() => {
-    nock(launcherWebserviceUri)
-      .get('/v1/Frameworks')
-      .reply(200, frameworkInfos)
-      .get('/v1/Frameworks')
-      .query({UserName: 'test1'})
-      .reply(200, {'summarizedFrameworkInfos': [frameworkInfos.summarizedFrameworkInfos[0]]});
-  });
-
   // GET /api/v1/jobs
-  it('should return job list', (done) => {
+  it('[P-01] Get job list', (done) => {
+    nock(launcherWebserviceUri)
+    .get('/v1/Frameworks')
```

Comment:
indentation

Reply:
please refer to indentation in [nock examples](https://github.com/node-nock/nock#specifying-request-body) or [line wrapping style](https://google.github.io/styleguide/jsguide.html#formatting-line-wrapping)